stanford-earth/SE3_BLT
# EARTH-16: Enable Full width banner paragraph type.

#
# ON HOLD

# WAITING FOR REVISIONS TO VARIANTS ON FWB

# Summary
- Enables FWB paragraph type.

# Needed By (Date)
- End of sprint

# Urgency
- Moderate

# Steps to Test

1. Check out this branch and do a full refresh (code & db)
2. Ensure FWB is enabled and available to complex page

# Associated Issues and/or People
- Just you.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
